src/Nri/Ui/AssignmentIcon/V1.elm

```diff
@@ -277,3 +278,232 @@ startSecondary =
+        ]
+        |> Nri.Ui.Svg.V1.fromHtml
```

Comment:
👍 this all looks great from what I can tell (based on what you demo'd to us today :) )